internal/database/db.go

```diff
@@ -9,8 +9,9 @@ import (
 
 const (
 	GetAllWorkoutsSql        = "SELECT * FROM workouts ORDER BY created_at DESC"
-	GetWorkoutsForUserSql    = "SELECT * FROM workouts WHERE accountid=$1 ORDER BY created_at DESC"
+	GetWorkoutsForUserSql    = "SELECT workouts.workoutid, workouts.workoutname, workouts.workoutdescription, workouts.created_at, workouts.media, workouts.last_updated, account.username, account.accountid FROM workouts JOIN account ON workouts.accountid = account.accountid WHERE username=$1 ORDER BY created_at DESC"
```

Comment:
reason for these columns returned instead of all *?

Reply:
In case we add more columns to the db at some point, I don't want us to be pulling data we don't need